src/pyodide/pyodide-bootstrap-helper.js

```diff
@@ -1,7 +1,5 @@
-import { loadPyodide } from "pyodide:python";
-import { getMetadata } from "pyodide:current-bundle";
-import { lockFile } from "pyodide:package-lock.json";
-import { getPatches } from "pyodide:patches";
+export { loadPyodide } from "pyodide-internal:python";
+import { lockFile } from "pyodide-internal:pyodide-lock";
```

Comment:
Since you're changing this anyway: i've been wondering why we include this file separately at all, it's part of the pyodide tarball, so why don't we just grab it from there?

Reply:
At an earlier point I had some hot patches for Pyodide bugs and they required changes to this file. All the changes got upstreamed into v0.25.0a2 though so it's no longer necessary.

Reply:
I'll plan to remove this in a followup (can probably be combined with update to Pyodide 0.25.0)